code/ArrayCount.java

```diff
@@ -3,10 +3,6 @@
 class ArrayCount {
   public static void main(String[] args) {
     List<Integer> arr = Arrays.asList(1, 11, 111);
-    int count = 0;
-    for (int n : arr)
-      if (n > 10)
-        count++;
-    System.out.println(count);
+    System.out.println(arr.stream().count());
```

Comment:
This is not just count, but filtered count. You should count only those values greater than 10. In Python this looks something like this:

```python
sum(1 for i in arr if i > 10)
```